### DISCUSSION TITLE
Cannot connect to macOS when Mac locked

### DISCUSSION THREAD

**Discussion by mrHJS** [rustdesk/rustdesk#7565] [General]
Hi there,
I set up my own RustDesk Server to remote work on my Mac when I am not home. This works as long as my Mac is unlocked and the screen is on. As soon as my Mac switches the screen off and locks the device, I cannot connect to my Mac any longer - RustDesk tells me, the device is offline.
Is this expected behavior? I don't want to leave my Mac unlocked all day but this want to be able to remotely connect to it. With RDP on Windows this works flawlessly.
Is there anything I can do about it?

Thank you!

**Comment by xlionjuan** [comment]
https://support.apple.com/en-us/guide/mac-help/mchle41a6ccd/mac

**Comment by FiftyKnight** [comment]
I have same problem.  My Mac mini is configured to not sleep.  When I lock the screen, or when I reboot, I cannot connect from RustDesk client from other computer.  I must be logged into the Mac and cannot lock the screen.  Ugh!
